src/foam/nanos/auth/Address.js

```diff
@@ -40,7 +40,7 @@ foam.CLASS({
     { name: 'INVALID_ADDRESS_1', message: 'Invalid value for address line 1' },
     { name: 'INVALID_POSTAL_CODE', message: 'Valid Postal Code or ZIP Code required' },
     { name: 'POSTAL_CODE_REQUIRE', message: 'Postal Code required' },
-    { name: 'STREET_NAME_REQUIRED', message: 'Street name required' },
+    { name: 'STREET_NAME_REQUIRED', message: 'Street Name required' },
```

Comment:
This message can be removed if you set required:true on the streetName field and remove the validationPredicates from it

Reply:
updated